GTHC/web
# Updated to Rails 5.2 and added avatar for users

**Warning**: _Since there has been an update to core services, please make sure that you follow the steps below when you are testing this PR and once this has been pushed to `staging`. If the steps are not followed, it is likely that development will not function for you locally._

## Setup (required)
As their has been an critical update to our rails dependency (as well as others). Please do the following:
1. Run `docker-compose down`
2. Run `docker-compose build`
3. Run `yarn flush`
4. Run `yarn dev:start` to start the server

## Description

Updated Rails 5.1 ~> Rails 5.2 in order to use new `ActiveStorage` library to upload files to models. In this PR, it focuses on both installing the ActiveStorage library and implementing it to the User model in order to allow users to have and upload avatars to their accounts. This also updates the availability dropdown to show user avatars.

## Motivation and Context


See trello card: https://trello.com/c/YySuYMuE/32-add-profile-picture-for-team-and-user-models

## How Has This Been Tested?



1. Go to User Profile, and see lack of profile picture.
2. Go to "Edit User Avatar", and upload and save an image.
3. Return to previous pane, and see that picture is now there.
4. Try to add or update shift, see that user availability dropdown now has all of the team mate's avatars.

## Screenshots (if appropriate):
![image](https://user-images.githubusercontent.com/13587407/50576780-418ee100-0de7-11e9-83f0-6e302ba2b873.png)

![image](https://user-images.githubusercontent.com/13587407/50576802-c4b03700-0de7-11e9-8b12-756bd9c28aa7.png)

![image](https://user-images.githubusercontent.com/13587407/50576819-1eb0fc80-0de8-11e9-91d5-769535fde37c.png)


![image](https://user-images.githubusercontent.com/13587407/50576815-06d97880-0de8-11e9-92c9-ce98fbab177d.png)

![image](https://user-images.githubusercontent.com/13587407/50579210-17521900-0e11-11e9-8f2e-c4e14dcd5d33.png)



## Types of changes


## Checklist:
